src/main/java/org/redcastlemedia/multitallented/civs/commands/ToggleChatChannelCommand.java

```diff
@@ -28,8 +30,10 @@ public boolean runCommand(CommandSender commandSender, Command command, String l
         T o = getRelevantTarget(civilian, args);
 
         LocaleManager localeManager = LocaleManager.getInstance();
-
-        if (isValid(civilian, o)) {
+    
+        ChatChannelConfig chatChannelConfig = ConfigManager.getInstance().getChatChannels().get(this.channelType);
+    
+        if (isValid(civilian, o) && chatChannelConfig.enabled) {
```

Comment:
I think this could throw npe if user had deleted specific chat channel from civs config.

Reply:
Yes, but i found out that civs brokes way before this if you deletes the channels from the config, it expects on load that those channels are defined